app/src/main/java/com/ryeslim/currencyexchange/CalculateCommission.kt

```diff
@@ -19,8 +19,10 @@ class SevenPercent : CalculateCommission {
         numberOfOperations: Int,
         extraCondition: Boolean
     ): BigDecimal {
-        return if (numberOfOperations > 5 && extraCondition) {
-            (amount * (0.7 / 100).toBigDecimal()).setScale(2, RoundingMode.HALF_EVEN)
+        val numberOfFreeOperations = 5
+        val commissionsPercent = 0.7
```

Comment:
`    val numberOfFreeOperations = 5
        val commissionsPercent = 0.7
`
Has to be in compianion object as constants